src/app/page/UserProfile/UserProfile.js

```diff
@@ -75,9 +76,39 @@ const UserProfile = ({ history }) => {
     setProfile(_.cloneDeep(firebaseProfile));
     setView("view");
   }
-  function saveProfileAction(e) {
+  async function saveProfileAction(e) {
     e.preventDefault();
-    firebase.updateProfile(profile);
+    try {
+      !profile.address
+        ? console.log(null)
+        : await addressLookUp(profile.address).then((data) =>
+            firebase.updateProfile({
+              ...profile,
+              addressMapPoint: { latitude: data.lat, longitude: data.long },
+            })
+          );
+    } catch (error) {
+      if (error.response) {
```

Comment:
Does there need to be any fallback action if we reach this catch? Like telling the user something?

Lets also clean up these console.logs

Reply:
I would use the ErrorSnackbar here.